homeassistant/components/mqtt/__init__.py

```diff
@@ -351,8 +361,8 @@ def async_setup(hass, config):
         return False
 
     # For cloudmqtt.com, secured connection, auto fill in certificate
-    if certificate is None and 19999 < port < 30000 and \
-       broker.endswith('.cloudmqtt.com'):
+    if (certificate is None and 19999 < port < 30000 and
```

Comment:
Please don't include these "style" fixes in PRs. It makes it difficult to see what this PR actually adds.

If you want to do those, do it in a separate PR. Just know that generally, style fixes best case will make the code more readable, worst case introduce bugs that didn't exist before.

Reply:
For this PR it's fine, just don't do it in the future please.

Reply:
Sure. I'll try to remember this.